camera_fx: 644.7825927734375
camera_fy: 644.0340576171875
camera_cx: 636.1322631835938
camera_cy: 367.4698486328125
